aws-janitor/resources/efs.go

```diff
@@ -123,12 +126,13 @@ func (ElasticFileSystems) ListAll(opts Options) (*Set, error) {
 }
 
 type elasticFileSystem struct {
-	ID string
+	Account string
+	Region  string
+	ID      string
 }
 
 func (efs elasticFileSystem) ARN() string {
-	// The current client library does not provide ARNs for file systems.
```

Comment:
I think this comment is actually false? (or maybe it was added at some point?) 

we should be able to use the ARN provided in the result: https://docs.aws.amazon.com/sdk-for-go/api/service/efs/#FileSystemDescription

Reply:
to use that, i've bumped the `github.com/aws/aws-sdk-go` dependency